.flowconfig

```diff
@@ -3,6 +3,8 @@
 .*/node_modules/editions/.*
 .*/node_modules/flow-coverage-report/.*
 .*/node_modules/preact/.*
+.*/node_modules/@storybook/.*
+.*/node_modules/radium/.*
```

Comment:
Guessing modern versions of Storybook use Radium?

Reply:
Not directly, but via `react-treebeard`. We use `glamorous` for our own components

Reply:
Oh cool! Wasn't aware you were a contributor to Storybook - thanks for your work there :)